fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaViewModel.java

```diff
@@ -189,12 +193,16 @@ public Single<MessageGeneral> deleteChirp(MessageID chirpId, long timestamp) {
   public Observable<List<Chirp>> getChirps() {
     return socialMediaRepository
         .getChirpsOfLao(laoId)
+        .observeOn(schedulerProvider.mainThread())
```

Comment:
What problem did this cause ?

Reply:
If this is not present, the whole computation will be done in the same thread as the one that published a new version, which was not the main thread.

This caused issues when trying to modify the views as this can only be done from the main thread.